README.md

```diff
@@ -48,7 +49,9 @@ $ docker run --name pdns \
   * `MYSQL_USER=root`
   * `MYSQL_PASS=root`
   * `MYSQL_DB=pdns`
+  * `MYSQL_DNSSEC=yes`
```

Comment:
Lets use `no` here cause the values indicates the defaults.

Reply:
Ok, clear